integration_tests/integration/prisonerReportReview.cy.ts

```diff
@@ -217,12 +230,79 @@ context('Prisoner report - reviewer view', () => {
     cy.visit(adjudicationUrls.prisonerReport.urls.review(12345))
     const PrisonerReportPage: PrisonerReport = Page.verifyOnPage(PrisonerReport)
 
+    PrisonerReportPage.reviewSummaryTitle().should('exist')
+    PrisonerReportPage.reviewSummary().should('not.exist')
     PrisonerReportPage.incidentDetailsSummary().should('exist')
     PrisonerReportPage.offenceDetailsSummary().should('exist')
     PrisonerReportPage.incidentStatement().should('exist')
     PrisonerReportPage.reportNumber().should('exist')
     PrisonerReportPage.reviewerPanel().should('exist')
   })
+  it('should contain the correct review summary - returned', () => {
```

Comment:
It's good that these tests are here but 2 things make it a maintenance problem:

 - All this is currently tested in the jest tests. That should be enough - adding more tests means more tests to change. You do need 1 int test to cover the review data, but that should suffice.
 - It's inconsistent with prisonerReport.cy.ts. i would expect both to have the same tests, or neither, or share a test somehow.